agent/config/ank-agent.conf

```diff
@@ -0,0 +1,46 @@
+# ank-agent.conf is the configuration file for an Ankaios agent.
+# This configuration file is formatted in the TOML language.
+
+# The version of the configuration file.
+version = 'v1'
+
+# The name to use for the registration with the server.
+# Every agent has to register with a unique name.
+# Agent name shall contain only regular upper and lowercase characters
+# (a-z and A-Z), numbers and the symbols "-" and "_".
+# name = 'agent_1'
+
+# The server url.
+# If started in insecure mode then the HTTP protocol shall be used,
+# otherwise the HTTPS protocol shall be used.
+# server-url = 'https://127.0.0.1:25551'
+
+# An existing path where to manage the fifo files.
+# run-folder = '/tmp/ankaios/'
```

Comment:
That is not clear to me. Has the user to create that folder? I have never done that.